flyway.groovy

```diff
@@ -0,0 +1,77 @@
+#!./lib/runner.groovy
+import com.gargoylesoftware.htmlunit.WebClient
+import com.gargoylesoftware.htmlunit.html.HtmlAnchor
+import com.gargoylesoftware.htmlunit.html.HtmlPage
+import hudson.util.VersionNumber
+import net.sf.json.JSONObject
+
+import java.util.regex.Pattern
+// Generates server-side metadata for Flyway command-line
+def webclient (){
+  def wc = new WebClient()
+  wc.javaScriptEnabled = false;
+  wc.cssEnabled = false;
+  return wc
+}
+
+def listFromMavenRepo() {
+  def json = []
+  def url = "https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/";
+
+  def wc=webclient();
+  HtmlPage p = wc.getPage(url);
+  def pattern = Pattern.compile("^([0-9][0-9\\.]+)/\$");
```

Comment:
Fair enough, I've committed a change that merely looks for links that begin with a number.